docs/src/main/tut/faq.md

```diff
@@ -24,6 +24,14 @@ position: 4
  * [What does this symbol mean?](#symbol)
  * [How can I help?](#contributing)
 
+## <a id="diff-scalaz" href="#diff-scalaz"></a>What is the difference between cats and scalaz? 
+
+The goal for both Cats and [Scalaz](https://github.com/scalaz/scalaz) are the same: facilitate pure functional programming in Scala applications; the underlying core strategy is different. Scalaz took the approach of trying to provide a single batteries-included *standard library* for FP that powers the Scala applications. Cats, on the other hand, aims to help build an [ecosystem](/cats/#ecosystem) of pure FP libraries by providing a solid and stable foundation. These libaries can have their own styles and personalities, competing with each other, while at the same time playing nice. It is through this ecosystem of FP libraries (cats included) that Scala applications can be powered with "FP awesome-ness" and beyond by picking whatever best fit their needs. 
+
+Based on this core strategy, Cats took a [modular](/cats/#modularity) approach and focuses on providing core, [binary compatible](/cats/#a-namebinary-compatibility-and-versioning), [approachable](/cats/#approachability) and [efficient](/cats/#efficiency) abstractions. It also takes great effort in providing a welcoming and supportive [community environment](https://gitter.im/typelevel/cats) and a comprehensive beginner-friendly [documentation](/cats/#documentation). 
```

Comment:
When discussing the supportive community, I think that it might make sense to point out that the community is governed by the code of conduct, as this is another difference from scalaz.